rotkehlchen/serialization/schemas.py

```diff
@@ -247,33 +296,65 @@ def make_custom_asset(
         return {'custom_asset': custom_asset}
 
 
-class AssetDataSchema(Schema):
+class AssetSchema(Schema):
     asset_type = AssetTypeField(required=True)
-    name = fields.String(required=True)
 
     class Meta:
         # Set unknown = 'INCLUDE' to allow extra parameters
         unknown = 'include'
 
+    def __init__(
+            self,
+            identifier_required: bool,
+            disallowed_asset_types: Optional[list[AssetType]] = None,
+            coingecko: Optional['Coingecko'] = None,
+            cryptocompare: Optional['Cryptocompare'] = None,
+            **kwargs: Any,
+    ) -> None:
+        super().__init__(**kwargs)
+        self.identifier_required = identifier_required
+        self.disallowed_asset_types = disallowed_asset_types
```

Comment:
Can you document in the docstring this two arguments? I bet that in two weeks I'll need to use those and I won't recall what I'm suppose to use